utils/Logger.py

```diff
@@ -37,9 +36,7 @@ def _should_log(log_type: DebugLevel):
     @staticmethod
     def _colorize_message(label, color, msg):
         date = datetime.now().strftime('[%d/%m/%Y %H:%M:%S]')
-        # No colors for Windows :)
-        if Logger.IS_WINDOWS:
-            return f'{label} {date} {msg}'
+        # Colorama seems to be working with windows nowadays.
```

Comment:
Did you test with both the `cmd` and PowerShell? Also in the embedded terminal of your IDE.